IMC_WebDev/MiniCityBlog.md

```diff
@@ -0,0 +1,96 @@
+
+Now, let’s make our classes! Let’s think about what we want our city to look like.
+We know that we want the following objects in our city:
+The ground
+A skyscraper
+A smaller building
+A road
+A bridge
+An ocean
+A waterfall
+A car
+
+In order to eventually create these objects, we have to make classes for them. Classes are basically like blank notebooks that we use to establish the fact that we are going to create an object. Then, we actually create it in CSS!
+
+We create classes using the "div" tag. Like this: <img src=https://lh6.googleusercontent.com/1iPgi651LuYWiyo5AnInaCkxrpUMpnVAeBFdxIiEtEpILCPIm1JixLkgK5oUWdsnLYMyOxsO2LjNdGGBz47F-g-yJpUt2dtV4mnathNFa6TGKwOJ2KJoXVRXrY2b9M0W6eQ6r6nR>
+As you can see, we just created a class called “ground”. Now our editor knows that we are planning to create an object.
+
+We do the same thing with all of our other objects. When you are done, it should look something like this: <img src=https://lh4.googleusercontent.com/KvgGx3EetIK7wY0oXVeEFvrvaznJTFkrwE6fwTi0HoYN4Rymi5SmJj6WAc-g2BUgAiFMjVm3WNUOdvusnb-0-cbewzMQDYZhIWOdrJRLtGd-KC2Ox1Ps7MVcq8Ev4bG5NOUirUz3> <img src=https://lh4.googleusercontent.com/QPwNwwgKTGt_I6I4pmBxnd5dvlew7OocTCup8YiqBB8DTmmnyaWZ2Qi5TK_GBboP6gjdJgObXWbt1-pIjPF95GlkQSGlZcfw0Z3Wa7amH-24IBkAMO4Z6mXmepX5uNoRb5kh9xao>
+
+
+This looks like a huge chunk of code right now, but let’s break it down!
+
+For every object in our city, we create a div class for it. That way, we can edit it using our CSS. HTML is just our canvas for all of our objects, while CSS is like our paint!
+
+Sometimes, objects have “children”. For example, since our buildings are placed on the ground, the buildings are “children” of the ground.
+You can see that “building1” is placed within the “ground” tag in our code.
+
+Our “building1” also has children of its own. Since walls and a roof are components of our building, we place them inside our “building1” tag. We repeat this whenever components belong to each other. (You can also see this with the grass area and trees)
+
+After we create all of our objects and their “children”, it is time to close our body and our html tag by using /body and /html.
+
+When it’s time to design our objects in CSS, we MUST use the exact name we created for them in our HTML. For example, if I want to change the color of the bridge in my city, I need to name it “bridge” in my CSS, just like how it is in my HTML.
+
+Next, let’s prepare our editor for the CSS portion of the code. We do this by establishing a style tag. Since we will be using CSS, our style tag will look like this: <img src=https://lh5.googleusercontent.com/kNUcMVEmTr0O0GyTUlYwlJro5lYXiehFh5gwS_B4xZujFUkrVZEn9pIjoA-venhZlkqoWjYBlksmDZ4o0-SEaem_FFR072jiqH26Svuy4ga0ACzWjt4m-5JQG1FI_mBGbIDppin0>
+
+We will be writing all of our CSS in our head portion of our code.
+
+To style our webpage, we will need to learn the CSS syntax. A CSS rule-set consists of a selector and a declaration block.
```

Comment:
Briefly explain what a selector, a declaration block, and even a rule-set is in this context in more detail.